en-us/Content/index-admin.htm

```diff
@@ -93,7 +93,7 @@
         <script>
 			/* <![CDATA[ */
 			(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-			new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+			new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0], 

```

Comment:
another example of many in the changes for this PR that shows that something changed in the formatting for some files (css, js) and that we should try to keep the format the same. Helps reduce the number of changes in each PR